test/spec/features/auto-place/AutoPlaceSpec.js

```diff
@@ -326,7 +456,7 @@ describe('features/auto-place', function() {
         });
 
         // when
-        var connectedDistance = getConnectedDistance(shape, 'x', null, {
+        var connectedDistance = getConnectedDistance(shape, {

```

Comment:
How do we handle different axis?

Reply:
You can pass a `direction` hint. It defaults to `w` (west).

Reply:
All right!